config/deploy_id_rsa_enc

```diff
@@ -0,0 +1,9 @@
+U2FsdGVkX1+qgmt3SzdcKx/FVhDrovWbFoetH5KIvym6073I2J0fhByPAY2svETO
```

Comment:
are you sure this should be in a commit?

Reply:
yup, this is the encrypted key that capistrano-deploy will use to connect to the server and pull the code from github